tools/buildsteps/windows/BuildSetup.bat

```diff
@@ -270,12 +270,14 @@ set WORKSPACE=%base_dir%\kodi-build
   call %base_dir%\project\Win32BuildSetup\extract_git_rev.bat > NUL
   for /F %%a IN ('dir /B /S %WORKSPACE%\AppPackages ^| findstr /I /R "%APP_NAME%_.*_%TARGET_ARCHITECTURE%_%buildconfig%\.appx$"') DO (
     copy /Y %%a %base_dir%\%APP_NAME%-%GIT_REV%-%BRANCH%-%TARGET_ARCHITECTURE%.appx
+    copy /Y %%a %base_dir%\%APP_NAME%-%GIT_REV%-%BRANCH%-%TARGET_ARCHITECTURE%.appxsym
```

Comment:
This copies appx to appxsym file instead of copying the appxsym file.

Reply:
arghhh